.github/workflows/releasability.yaml

```diff
@@ -26,9 +26,6 @@ on:
       slackChannel:
         type: string
         required: false
-    secrets:
-      SLACK_WEBHOOK:
-        required: true
```

Comment:
Secrets still need to be passed explicitly 

https://docs.github.com/en/actions/using-workflows/reusing-workflows#example-reusable-workflow